app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,22 @@
+module SessionsHelper
+
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
```suggestion
      @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Reply:
if you using this code, you should delete line 7 and 9 
just only need @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]